services/dis-apim-operator/internal/azure/apim_client.go

```diff
@@ -0,0 +1,127 @@
+package azure
```

Comment:
Just a comment, not a review: I guess this one could potentially be moved to a pkg folder later, since it sounds like it could be re used in some other apps and projects.